### PR TITLE
Admin Export Template Image: Add Ops, TA Cmdlet and security check fo…

### PR DESCRIPTION
**Pull request by arcadiahlyy**
…r exporting template image out of ARA

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@arcadiahlyy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hele). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by shahabhijeet** [comment]
@arcadiahlyy I don't see any tests for the new functionality that you have added.

**Comment by arcadiahlyy** [comment]
Will open a new one with updates.

**Comment by azurecla** [comment]
Hi __@arcadiahlyy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hele). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by arcadiahlyy** [comment]
The old PR got updated, so reopened it.

File: src/ServiceManagement/RemoteApp/RemoteApp/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -45,4 +45,4 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("2.0.0.0*")]
 [assembly: AssemblyVersion("2.0.0.0")]
```
Comment (arcadiahlyy) [inline src/ServiceManagement/RemoteApp/RemoteApp/Properties/AssemblyInfo.cs]: This has been fixed. Thanks.

**Pull request merged**